CloudNotes/CloudNotes/View/Cell/NoteListCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  NoteListCellTableViewCell.swift
+//  CloudNotes
+//
+//  Created by 윤재웅 on 2021/06/03.
+//
+
+import UIKit
+
+class NoteListCell: UITableViewCell {
+    var noteData: NoteData? {
+        didSet {
+            displayData()
+        }
+    }
```

Comment:
이 프로퍼티를 외부에서 알아야할까요? 혹시 값을 주입해주기 위해 알아야한다면, 직접 접근하기보다 인터페이스를 두는게 좋을 것 같습니다.

```suggestion
    private var noteData: NoteData?
    
    func bind(noteData: NoteData?) {
        displayData()
    }
```

Reply:
```swift
// cellForRowAt
cell.displayData(noteListManager.dataAtIndex(indexPath.row))

// cell 
func displayData(_ data: Note) {
    titleLabel.text = data.title
    dateLabel.text = data.date
    descriptionLabel.text = data.description
}
```

따로 저장하지 않고 화면에 바로 올려주도록 수정했습니다